home-screen-data-source/src/types.d.ts

```diff
@@ -0,0 +1,22 @@
+declare namespace Express {
+    export interface Request {
+        auth?: JWTPayload;
+    }
+}
+
+
+interface JWTPayload {
```

Comment:
The properties here should have some docstrings, as their names are so much shortened that it's difficult to tell what values they will hold.

Reply:
They are rather standard. I added docstrings though.